Algorithm.CSharp/CustomBenchmarkRegressionAlgorithm.cs

```diff
@@ -83,18 +83,18 @@ public override SubscriptionDataSource GetSource(SubscriptionDataConfig config,
             {"Drawdown", "2.300%"},
             {"Expectancy", "0"},
             {"Net Profit", "0.382%"},
-            {"Sharpe Ratio", "5.488"},
-            {"Probabilistic Sharpe Ratio", "60.047%"},
+            {"Sharpe Ratio", "1.334"},
+            {"Probabilistic Sharpe Ratio", "50.998%"},
             {"Loss Rate", "0%"},
             {"Win Rate", "0%"},
             {"Profit-Loss Ratio", "0"},
-            {"Alpha", "-0.001"},
-            {"Beta", "0.997"},
-            {"Annual Standard Deviation", "0.179"},
-            {"Annual Variance", "0.032"},
-            {"Information Ratio", "-7.724"},
-            {"Tracking Error", "0"},
-            {"Treynor Ratio", "0.986"},
+            {"Alpha", "-1.196"},
+            {"Beta", "0.777"},
```

Comment:
`"Beta", "0.997"` -> `"Beta", "0.777"` Minute AAPL ? 😬

Reply:
Same as above with BasicTemplateDaily

>Has to do with the order being executed at market open.
>First benchmark value is day 1 at 12AM, because we buy during day 1 at open, the price will have changed. Then the performance for day 1 benchmark and equity will be off.